src/imagelightbox.js

```diff
@@ -440,20 +531,28 @@
                 image = $();
             },
 
-            _openImageLightbox = function ($target) {
+            _openImageLightbox = function ($target, noHistory) {
```

Comment:
is noHistory necessary as a param?

Reply:
It is necessary to have the value somehow to distinguish between the lightbox opening by user action and by history popback. Param seems to me to be the best way (no worries about globals to be set and having to unset them).